cmd/version.go

```diff
@@ -8,26 +8,38 @@ import (
 )
 
 func versionCmd() *cobra.Command {
-	return &cobra.Command{
+	var silent bool
+
+	command := cobra.Command{
 		Use:   "version",
 		Short: "Print version/build info",
 		Long:  "Print version/build information",
 		Run: func(cmd *cobra.Command, args []string) {
-			printVersion()
+			printVersion(silent)
 		},
 	}
+
+	command.PersistentFlags().BoolVarP(&silent, "short", "s", false, "Simplified print version for resumed printing")
+
+	return &command
 }
 
-func printVersion() {
+func printVersion(silent bool) {
 	const secFmt = "%-10s "
+	var outputColor color.Paint
 
-	printLogo(color.Cyan)
-	printTuple(secFmt, "Version", version)
-	printTuple(secFmt, "Commit", commit)
-	printTuple(secFmt, "Date", date)
+	if silent {
+		outputColor = color.White
```

Comment:
@brunohms use outputColor = color.None or something like that to tell the colorizer not to colorize the output. See comment below...

Reply:
Changed it on my other commit, however it wasn't to color.None, as color is an int.
Set it to `-1` and checked if outputColor is -1, if it is, I do no print colors.
Check using vim.

Output would look like this:
```
Version     dev
Commit      dev
Date        n/a
```

Reply:
Sounds good. Tx Bruno!!